docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -159,36 +164,102 @@ but it should become "Bound" fairly quickly.
+``mongo-configdb-claim`` PersistentVolumeClaims for its ``/data/db`` and
+``/data/configdb`` diretories (mount path).
+
+Note also that we use the pod's ``securityContext.capabilities.add`` specification
+to add the ``FOWNER`` capability to the container.
+
+That is because MongoDB container has the user ``mongodb``, with uid ``999``
+and group ``mongodb``, with gid ``999``.
+When this container runs on a host with a mounted disk, the writes fail when
+there is no user with uid ``999``.
+
+To avoid this, we use the Docker feature of ``--cap-add=FOWNER``.
+This bypasses the uid and gid permission checks during writes and allows data
+to be persisted to disk.
+Refer to the
+`Docker doc <https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities>`_
+for details.
 
-Take a look inside that file to see how it defines the Service
-and the StatefulSet.
-Note how the MongoDB container uses the ``mongoclaim`` PersistentVolumeClaim
-for its ``/data`` diretory (mount path).
+As we gain more experience running MongoDB in testing and production, we will
+tweak the ``resources.limits`` values for cpu and memory.
+We will also stop exposing port ``27017`` globally and/or allow only certain
+hosts to connect to the MongoDB instance in the future.
 
-Create the StatefulSet and Service in your cluster using:
+Create the required StatefulSet using:
 
 .. code:: bash
 
-   $ kubectl apply -f node-mdb-ss.yaml
+   $ kubectl apply -f mongo-ss.yaml
 
-You can check that they're working using:
+You can check its status using the commands ``kubectl get statefulsets -w``
+and ``kubectl get svc -w``
+
+ 
+Step 6: Run BigchainDB as a Deployment
+--------------------------------------
+
+Get the file ``bigchaindb-dep.yaml`` from GitHub using:
+
+.. code:: bash
+
+   $ wget https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/k8s/bigchaindb/bigchaindb-dep.yaml
+
+Note that we set the ``BIGCHAINDB_DATABASE_HOST`` to ``mdb`` which is the name
+of the MongoDB service defined earlier.
+
+We also hardcode the ``BIGCHAINDB_KEYPAIR_PUBLIC``,
+``BIGCHAINDB_KEYPAIR_PRIVATE`` and ``BIGCHAINDB_KEYRING`` for now.
+
+As we gain more experience running BigchainDB in testing and production, we
+will tweak the ``resources.limits`` values for cpu and memory, and as richer
+monitoring and probing becomes available in BigchainDB, we will tweak the
+``livenessProbe`` and ``readinessProbe`` parameters.
+
+We also plan to specify scheduling policies for the BigchainDB deployment so
+that we ensure that BigchainDB and MongoDB are running in separate nodes, and
+build security around the globally exposed port ``9984``.
+
+Create the required Deployment using:
 
 .. code:: bash
 
-   $ kubectl get services
-   $ kubectl get statefulsets
+   $ kubectl apply -f bigchaindb-dep.yaml
+
+You can check its status using the command ``kubectl get deploy -w``
+
+
+Step 7: Verifying our node is setup correctly
```

Comment:
Titles should be written in title case, and should take an imperative "Do This" form, e.g. "Step 7: Verify the BigchainDB Node Setup"